src/editor/actions/__tests__/fetch-cell-parser.test.js

```diff
@@ -132,7 +132,7 @@ const validFetchLines = [
     line: "css: files/company-report-styles.css  // comment",
     result: {
       fetchType: "css",
-      filePath: "files/company-report-styles.css",
+      filePath: "company-report-styles.css",
```

Comment:
note that the input line is still "css: files/company-report-styles.css  // comment", so we're still testing inputs beginning with "files/" -- the proposed change just parses that out (which we were actually already doing in `extractFileNameFromLocalFilePath`), and prepends `files/` in the function `loadFileFromServer` so that communication with the iodide server API works ok (since that the "files/" path is expected by the server).